cura/ApplicationMetadata.py

```diff
@@ -42,6 +42,11 @@
 except ImportError:
     CuraDebugMode = DEFAULT_CURA_DEBUG_MODE
 
+try:
+    from cura.CuraVersion import CuraIsSecuredVersion  # type: ignore
+except ImportError:
+    CuraIsSecuredVersion = (CuraBuildType.lower() in ["essentials", "enterprise", "assured", "secure", "secured"])
```

Comment:
Missing 'shampoo' in this list :P

Reply:
... you know, I had the same thought when I was driving home after writing this :-)